app/src/main/java/org/permanent/permanent/ui/login/LoginFragment.kt

```diff
@@ -53,6 +55,7 @@ class LoginFragment : PermanentBaseFragment() {
     }
 
     private val onLoggedIn = Observer<Void> {
+        logEvents()
```

Comment:
There are multiple places from where the user is logged in. Maybe should add this there too.